src/components/PicklistOption/index.js

```diff
@@ -0,0 +1,179 @@
+/* eslint-disable no-script-url, react/prop-types, jsx-a11y/anchor-is-valid */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { Consumer } from '../Picklist/context';
+import Icon from './icon';
+import './styles.css';
+
+class Option extends Component {
+    constructor(props) {
+        super(props);
+        this.itemRef = React.createRef();
+        this.handleClick = this.handleClick.bind(this);
+        this.handleHover = this.handleHover.bind(this);
+    }
+
+    componentDidMount() {
+        const { privateRegisterChild, disabled, variant, label, name, icon, value } = this.props;
+        const isHeader = variant === 'header';
+        if (disabled || isHeader || typeof name !== 'string') {
+            return null;
+        }
+        return setTimeout(
+            () =>
+                privateRegisterChild(this.itemRef.current, {
+                    label,
+                    name,
+                    icon,
+                    value,
```

Comment:
here only send what you need, I believe you only need the name

Reply:
it is necessary to send all because in PickList.handleKeyEnterPressed we have to fire an onChange event

Reply:
👍